pkg/operators/localmanager/localmanager.go

```diff
@@ -370,9 +370,24 @@ func (l *localManagerTrace) PreGadgetRun() error {
 		}
 
 		if host {
+			pidOne := uint32(1)
+			mntns, err := containerutils.GetMntNs(int(pidOne))
+			if err != nil {
+				return fmt.Errorf("getting mount namespace ID for host PID %d: %w", pidOne, err)
+			}
```

Comment:
In general, if it is used in the same way for all gadgets, you could do it in `Init()` of the operator itself. If it's something that is unique to each gadget run, it should be done on `Instantiate()`. So in this case, I think it could be done in `Init()` and then forwarded to the instance (in `Instantiate()`) or directly accessed using `l.manager.fakeHostContainer` or such.

Reply:
In general, I see two main disadvantages in keeping such things globally:

* it could make tests harder (in this specific case for example if you want to have an operator instance with support for the fake container and one without)
* if you do stuff in `init()`, it could lead to longer start up times and nondeterministic initialization order (since it's built around the dependency chain)

In this case it would probably be okay since both arguments don't really matter here, but I'd still prefer having it as member of the operator.